keras_retinanet/bin/evaluate.py

```diff
@@ -116,7 +119,13 @@ def parse_args(args):
     parser.add_argument('--save-vector',      help='List of geometry types to save vector.', nargs='+', type=str, default=[], choices=['polygon', 'point'])
     parser.add_argument('--weighted-average', help='Compute the mAP using the weighted average of precisions among classes.', action='store_true')
```

Comment:
In evaluate.py do I need to change text in prints so that it will be clear is it weighted number? And in callbacks/eval.py icalc_class_precisions returns only one  value which is printed and shown on tensorboard. What should we do in case this function reruns 2 values?

Reply:
Yes, I saw these problems. Because of them I decided to groom it and solve it as another task. Just create a task in Jira, write description and copy text of this discussion to comments of task. With it you may resolve this converation and all other are touched to this task.